Makefile

```diff
@@ -274,6 +280,10 @@ controller-gen: $(LOCALBIN)
 envtest: $(LOCALBIN)
 	@test -x $(ENVTEST) || GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@$(ENVTEST_VERSION)
 
+.PHONY: crd-ref-docs
+crd-ref-docs: $(LOCALBIN)
+	@test -x $(CRD_REF_DOCS) || GOBIN=$(LOCALBIN) go install github.com/elastic/crd-ref-docs@latest
```

Comment:
I think it's a good pattern to define a particular version where it's possible.
For inspiration take a look here:

https://github.com/aenix-io/etcd-operator/blob/cc22fbd58659b9cb6a225e825c4eb6c30e2247ea/Makefile#L316C1-L319C95

Reply:
Sadly it doesn't provide any way to fetch version from binary